### ISSUE TITLE
switch to micromatch?

### ISSUE THREAD

**Issue by vvo** [carrot/globtester#10]
It seems like https://github.com/micromatch/micromatch is the new cool kid around the block, we are experimenting with it in a new project and like the API a lot. Would you like to move this nice globtester tool to it?

**Comment by jescalan** [comment]
Yeah I would actually. If you'd like you're welcome to PR this -- if not I'll do it as soon as I have a few moments.

**Comment by vvo** [comment]
Not sure I am able to easily do that with coffee script. But I like the idea of having a regexp like tester for globs. Much needed and I would use it a lot if:
- was using micromatch (and allows for multi lines patterns, like the lib does)
- was providing some cheatsheet on the side (based on the various glob patterns we can found online)

This is a great idea and worth some fame on hacker news I am sure :)

**Comment by jescalan** [comment]
Haha I understand, but this is open source, where people collaborate, not ask other people to build things for them. Also, you have the ideas, so why not be part of the fame?

How about this -- I'll convert this to es6 right now (this should be done anyway) then maybe you can jump in and see about contributing? 😁

**Comment by vvo** [comment]
That would help :)
